src/utils.py

```diff
@@ -0,0 +1,17 @@
+def parse_msg_json(msg):
+  """This parses the json string from the message field.
+  """
+  start_idx = msg.find('{')
+  if start_idx == 1:
+    return ''
+  return msg[start_idx:]
+
+def parse_key(msg_json, key):
+  """This parses the value corresponding to [key] from the msg_json.
+  """
+  key_idx = msg_json.find(key)
+  if key_idx == -1:
+    return ''
+  key_start_idx = key_idx + len(key) + 3
```

Comment:
should you make `3` a constant somewhere or is this too trivial to do that bc i have no idea why you need to add 3